erpnext/hr/doctype/regular_work_summary_group/regular_work_summary_group.json

```diff
@@ -0,0 +1,175 @@
+{
+ "autoname": "Prompt",
+ "creation": "2019-06-01 20:33:50.300725",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "enabled",
+  "select_users",
+  "users",
+  "section_break_holidays",
+  "send_emails_frequency",
+  "holiday_list",
+  "column_request",
+  "send_request_emails_at",
+  "send_request_emails_week_day",
+  "send_request_emails_month_day",
+  "send_request_emails_month",
+  "column_summary",
+  "send_summary_emails_at",
+  "send_summary_emails_week_day",
+  "send_summary_emails_month_day",
+  "send_summary_emails_month",
+  "section_email_content",
+  "subject",
+  "message"
+ ],
+ "fields": [
+  {
+   "default": "1",
+   "fieldname": "enabled",
+   "fieldtype": "Check",
+   "label": "Enabled"
+  },
+  {
+   "fieldname": "select_users",
+   "fieldtype": "Section Break",
+   "label": "Select Users"
+  },
+  {
+   "fieldname": "users",
+   "fieldtype": "Table",
+   "label": "Users",
+   "options": "Regular Work Summary Group User",
+   "reqd": 1
+  },
+  {
+   "fieldname": "section_break_holidays",
+   "fieldtype": "Section Break"
+  },
+  {
+   "fieldname": "holiday_list",
+   "fieldtype": "Link",
+   "label": "Holiday List",
+   "options": "Holiday List"
+  },
+  {
+   "default": "What did you work on today?",
+   "fieldname": "subject",
+   "fieldtype": "Data",
+   "label": "Subject"
+  },
+  {
+   "default": "<p>Please share what did you do today. If you reply by midnight, your response will be recorded!</p>",
+   "fieldname": "message",
+   "fieldtype": "Text Editor",
+   "label": "Message"
+  },
+  {
+   "default": "Daily",
+   "fieldname": "send_emails_frequency",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Send Reminder",
+   "options": "Daily\nWeekly\nMonthly\nYearly",
+   "reqd": 1
+  },
+  {
+   "fieldname": "send_request_emails_at",
+   "fieldtype": "Select",
+   "label": "Time",
+   "options": "00:00\n01:00\n02:00\n03:00\n04:00\n05:00\n06:00\n07:00\n08:00\n09:00\n10:00\n11:00\n12:00\n13:00\n14:00\n15:00\n16:00\n17:00\n18:00\n19:00\n20:00\n21:00\n22:00\n23:00"
+  },
+  {
+   "default": "Friday",
```

Comment:
Any particular reason to set Friday as default Day for sending Regular Work Summary ?

Reply:
Just had an idea that Friday is likely to be a  common option for weekly reports... Do you have any policy for this sort of choices?

Reply:
Let the user decide which day, make it mandatory and remove the default.

Reply:
That would be wrong to make it mandatory because it's not always visible. If you choose Daily, Monthly or Yearly you don't see the week day field. Will set to Monday both fields.

Reply:
Unless mandatory field constraint is checked against `eval:doc.send_emails_frequency == "Weekly";` and does't raise an error for empty field or Select field can not be empty which would mean some sort of default value...